python/tvm/topi/sparse/utils.py

```diff
@@ -116,11 +170,187 @@ def convert_model_dense_to_sparse(mod, params, random_params=False, bs_r=1, bs_c
     Dict[Srting, tvm.nd.array]
         The updated parameters.
     """
+
     mod, params = ddo.simplify_fc_transpose.convert(mod["main"], params)
     if random_params:
         # Manually replace the parameters of dense model to sparse tensors
         params = random_sparse_dense_params(mod, params, bs_r=bs_r, bs_c=bs_c, density=1 - sparsity)
+        params = random_sparse_conv2d_params(
+            mod, params, bs_r=bs_r, bs_c=bs_c, density=1 - sparsity, layout=layout
+        )
     # Currently we only support to conver dense matmul to sparse dense matmul
```

Comment:
Delete this line or add description about Conv2d.

Reply:
Done.